py/selenium/webdriver/support/wait.py

```diff
@@ -16,15 +16,17 @@
 # under the License.
 
 import time
+import typing
+
 from selenium.common.exceptions import NoSuchElementException, InvalidSelectorException
 from selenium.common.exceptions import TimeoutException
 
-POLL_FREQUENCY = 0.5  # How long to sleep in between calls to the method
-IGNORED_EXCEPTIONS = (NoSuchElementException,)  # exceptions ignored during calls to the method
+POLL_FREQUENCY: float = 0.5  # How long to sleep in between calls to the method
+IGNORED_EXCEPTIONS: typing.Iterable[typing.Type[Exception]] = (NoSuchElementException,)  # default to be ignored.
```

Comment:
this is technically a `typing.Tuple`; will change; its the more `ignored_exceptions` in the wait initialiser that would be of this type